# -- BEGIN LICENSE BLOCK ----------------------------------------------
# -- END LICENSE BLOCK ------------------------------------------------

TrackedObject object
geometry_msgs/Pose2D speed
float32 height
float32 heightErrorCovariance
